core/templates/dev/head/app.js

```diff
@@ -334,6 +334,15 @@ oppia.factory('validatorsService', [
   };
 }]);
 
+// Service for generating random IDs.
+oppia.factory('IdGenerationService', [function() {
```

Comment:
Nope -- we're moving towards CamelCase for everything in the frontend:

https://docs.google.com/document/d/1ZDmLN66f53WdDPItFChu9Lr37z0dKoqR-ASX8UM5y60/edit

Great that you spotted this, though! I realize it looks inconsistent; we need to finish the rest of that migration.